pepper-apis/pom.xml

```diff
@@ -521,7 +521,13 @@
             <dependency>
                 <groupId>org.testcontainers</groupId>
                 <artifactId>mysql</artifactId>
-                <version>1.17.6</version>
+                <version>1.18.1</version>
```

Comment:
Looks like we set the mysql version via `DatabaseTestUtil.MYSQL_VERSION`, separate from the version of testcontainers, so no concerns here.